### PR TITLE
initial version of custom commit tree creation

### PR DESCRIPTION
**Pull request by janfjohannes**
@martin-lysk  the test is still a placeholder, could you review the rest and double check i merged your changes correctly?

### CODE REVIEW

**Review by martin-lysk — APPROVED** [review]

**Comment by felixhaeberle** [comment]
@janfjohannes status? :)

**Comment by janfjohannes** [comment]
@felixhaeberle could we avoid status pings in comments? what is the motivation maybe we can change something to make something clearer?
Screenshot 2023-11-24 at 15 12 27

File: lix/source-code/fs/package.json [hunk 1]
```diff
@@ -6,28 +6,27 @@
 		"access": "public"
 	},
 	"exports": {
-		".": "./dist/index.js"
+		".": "./dist/index.js",
+		"./src/*.ts": "./src/index.d.ts"
```
Comment (janfjohannes) [inline lix/source-code/fs/package.json:10]: remove

**Review by samuelstroschein — APPROVED** [review]

**Comment by janfjohannes** [comment]
blocked by #1296

**Comment by janfjohannes** [comment]
fiiiiiiiinally green

**Pull request merged**